hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -10529,6 +10529,181 @@ objects:
+                      preferredTimer:
+                        hpet:
+                          present: false
+                        hyperv: {}
+                        pit:
+                          tickPolicy: delay
+                        rtc:
+                          tickPolicy: catchup
+                    cpu:
+                      dedicatedCpuPlacement: true
+                      preferredCPUTopology: sockets
+                    devices:
+                      preferredAutoattachInputDevice: true
+                      preferredDiskBus: virtio
+                      preferredInputBus: virtio
+                      preferredInputType: tablet
+                      preferredInterfaceModel: virtio
+                      preferredTPM:
+                        persistent: true
+                    features:
+                      preferredAcpi: {}
+                      preferredApic: {}
+                      preferredHyperv:
+                        frequencies: {}
+                        ipi: {}
+                        reenlightenment: {}
+                        relaxed: {}
+                        reset: {}
+                        runtime: {}
+                        spinlocks:
+                          spinlocks: 8191
+                        synic: {}
+                        synictimer:
+                          direct: {}
+                        tlbflush: {}
+                        vapic: {}
+                        vpindex: {}
+                      preferredSmm: {}
+                    firmware:
+                      preferredEfi:
+                        persistent: true
+                        secureBoot: true
+                    preferredTerminationGracePeriodSeconds: 3600
+                    requirements:
+                      cpu:
+                        guest: 2
+                      memory:
+                        guest: 4Gi
+              - complianceType: mustonlyhave
+                metadataComplianceType: musthave
+                objectDefinition:
+                  apiVersion: admissionregistration.k8s.io/v1
+                  kind: ValidatingAdmissionPolicy
+                  metadata:
+                    name: windows-vcpu-overcommit
+                  spec:
+                    failurePolicy: Fail
+                    matchConditions:
+                    - expression: (('kubevirt.io/preference-name' in object.metadata.annotations)
+                        && (object.metadata.annotations['kubevirt.io/preference-name'].lowerAscii().contains('windows')))
+                        || (('kubevirt.io/cluster-preference-name' in object.metadata.annotations)
+                        && (object.metadata.annotations['kubevirt.io/cluster-preference-name'].lowerAscii().contains('windows')))
+                        || (('vm.kubevirt.io/os' in object.metadata.annotations) &&
+                        (object.metadata.annotations['vm.kubevirt.io/os'].lowerAscii().contains('windows')))
```

Comment:
I might be misunderstancing KubeVirt preferences, but just in case an existing customer is using kubevirt for something that has "windows" in the name but isn't Windows OS, can we make sure some public-facing documentation warns the customer not to put "windows" in their preference names? I want to make sure our customer service agents have something to point to if a customer complains about their unfortunately-named non-windows VMs not scheduling anymore.

Reply:
@abyrne55 which documentation might be suitable?